### PR TITLE
Add encodeURIComponent() - jaak

### PR DESCRIPTION
**Pull request by yeeha567**
Test pull request

### CODE REVIEW

**Comment by yeeha567** [comment]
Hello.. You there. Take alook.

File: testJs.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 // This is my first Javascript code!
 console.log("Hello World");
 
-
+console.log("Hello not world");
```
Comment (boheman) [inline testJs.js:4]: Typo: s/Hello/Hell/

File: testJs.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 // This is my first Javascript code!
```
Comment (boheman) [inline testJs.js:1]: Delete this comment

**Review by boheman — CHANGES_REQUESTED** [review]

File: testJs.js (hunk 2 above)
Comment (yeeha567) [reply testJs.js:1]: Done.

File: testJs.js (hunk 1 above)
Comment (yeeha567) [reply testJs.js:4]: Done.

**Comment by boheman** [comment]
K. Look fine.

**Comment by yeeha567** [comment]
Approve?

**Comment by boheman** [comment]
wah?

**Pull request merged**